Solutions/UEBA Essentials/Hunting Queries/terminatedEmployeeAccessHVA.yaml

```diff
@@ -9,13 +9,17 @@ requiredDataConnectors:
 tactics:
 relevantTechniques:
 query: |
+  //Consider creating the watchlist named "TerminatedEmployees" and "HighValueAssets" having column "User Principal Name" and "Asset Name" with details of terminated employees
+  //If you already have watch list with diffrent name then consider modifying the query
```

Comment:
//If you already have watchlist with different name then consider modifying the query